src/ui/zabapgit_css_theme_dark.w3mi.data.css

```diff
@@ -199,6 +199,7 @@ div.log { color: var(--theme-greyscale-dark); }
 .dialog li.dialog-commands input[type="submit"] {
   border-color: #ccc;
   background-color: var(--theme-greyscale-dark);
+  color: #fff;
```

Comment:
Maybe add it as a var ? something like `button-caption-color` ?
Otherwise all looks good.

Reply:
will do it as part of a CSS rework. fabian had started https://github.com/abapGit/abapGit/pull/3956. something for 2023 to do